Sources_v3/StreamChatUI/ChatChannelList/ChatChannelListCollectionViewCell.swift

```diff
@@ -5,4 +5,62 @@
 import Foundation
 import UIKit
 
-open class ChatChannelListCollectionViewCell: UICollectionViewCell {}
+open class ChatChannelListCollectionViewCell<ExtraData: UIExtraDataTypes>: UICollectionViewCell {
+    // MARK: - Properties
+    
+    public weak var collectionView: UICollectionView?
```

Comment:
We have a **collectionView: UICollectionView?** instead of **collectionViewSize: CGSize?** to make the `layout` accessible in overridden `preferredLayoutAttributesFitting` in cell subclasses.

Reply:
fixed